pkg/apis/batch/validation/validation.go

```diff
@@ -384,9 +386,43 @@ func validateTimeZone(timeZone *string, fldPath *field.Path) field.ErrorList {
 		allErrs = append(allErrs, field.Invalid(fldPath, timeZone, err.Error()))
 	}
 
+	if err := validateTimeZoneCase(*timeZone); err != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath, timeZone, err.Error()))
+	}
+
 	return allErrs
 }
 
+// validateTimeZoneCase compares the timeZone with golang's builtin database
+// to ensure the caseing is correctly recognized. This only applies to MacOS
```

Comment:
maybe I am missing something, but I a bit confused when I read this description.

Reply:
I've improved the description a bit.